lib/kitchen/verifier/busser.rb

```diff
@@ -46,7 +46,7 @@ class Busser < Kitchen::Verifier::Base
         if verifier.windows_os?
           "$env:systemdrive\\opscode\\chef\\embedded\\bin"
         else
-          "/opt/chef/embedded/bin"
+          verifier.remote_path_join(%W[#{verifier[:chef_omnibus_root]} embedded bin])
```

Comment:
You can think of `config[:chef_omnibus_root]` as config that only comes from Chef-derived Provisioner plugins and don't leak into Driver, Transport, or Verifier plugins (re: Busser is a Verifier).

I may have a play with this to see if there's a way to get the *chef* part of out Busser if possible (it wouldn't apply to shell-only Provisioners for example that still would use Busser).

@scotthain We had talked through a config feature called `common` (or something like that) that would copy its data into all plugins at the same level--a way to lazily inject `config[:http_proxy]` into Provisioner and Verifier configs. We could use something like this here. Anyway, I'll update this PR with more